test/parallel/test-http2-respond-file-range.js

```diff
@@ -14,7 +15,7 @@ const {
   HTTP2_HEADER_LAST_MODIFIED
 } = http2.constants;
 
-const fname = path.resolve(common.fixturesDir, 'printA.js');
+const fname = path.resolve(fixturesDir, 'printA.js');
```

Comment:
Nit: this could be `fixtures.fixturesPath('printA.js')`.

Reply:
Well, it would be `fixtures.path()`